### ISSUE TITLE
"TaskForce" missing in help

### ISSUE THREAD

**Issue by jaymanu79** [sma/conquest#1]
Hello
I'm happy not being alone remembering that old source code.
It seems that you missed out "TaskForce" word from the original source :
```
conquest.dart:2594:  Helpst("D", " Destination"), conqs3.c:195:        {       "D",    "TaskForce Destination"  },
conquest.dart:2597:  Helpst("J", "Join s"),       conqs3.c:198:        {       "J",    "Join TaskForces"        },
conquest.dart:2600:  Helpst("N", "New "),         conqs3.c:201:        {       "N",    "New TaskForce"          },
conquest.dart:2604:  Helpst("T", " summary")      conqs3.c:205:        {       "T",    "TaskForce summary"      },
conquest.dart:2615:  Helpst("T", " summary"),     conqs3.c:217:        {       "T",    "TaskForce summary"      },
conquest.dart:2629:  Helpst("T", " summary")      conqs3.c:232:        {       "T",    "TaskForce summary"      },
```

**Comment by sma** [comment]
Wow, I never thought that someone would look at the source code :-) Thanks for noticing.

**Comment by jaymanu79** [comment]
Well I never thought that someone would port this to another language :)
I don't know what dart is, but actually the c code can still be compiled with gcc, using -std=c89, with only a few minor changes.

**Comment by sma** [comment]
> Well I never thought that someone would port this to another language :)

Some 22 years ago, I recreated it in [Smalltalk](http://3plus4.de/squeak/). I played the game in 1987 and that must have left quite an impression. However, I never researched its origin. I'm pretty sure that the C version from that Fish disk was also a port from a different language and the game is even older.

> I don't know what dart is, but actually the c code can still be compiled with gcc, using -std=c89, with only a few minor changes.

[Dart](https://dart.dev/) is Google's attempt to create a somewhat modern Java-like language nowadays mainly used to create cross platform UIs using the [Flutter](https://flutter.dev/) framework (which is what I do for a living).

**Comment by jaymanu79** [comment]
Well unfortunately Fred Fish passed away.. Maybe we could get some historical facts if we could find that Bob Shimbo on the internet.
i adapted it to ncurses, and just found that it has already been done :) https://github.com/beejjorgensen/conquest
Are you aware of that later version ? https://www.youtube.com/watch?v=Kgi6WSiMYUQ

**Comment by sma** [comment]
Yes, I'm aware of that code but never checked it out. Many years ago, created my own C port. I also have an unpublished Swift version of that game ;-) I published the Dart version because I talked about it and wanted to share it.

I think, Rob is short for [Robert](https://www.mobygames.com/developer/sheet/view/developerId,281903/) and perhaps it is [this guy|(https://www.officialusa.com/names/Robert-Shimbo/). I just googled for a few minutes.
